yax/state/jones.py

```diff
@@ -46,8 +75,9 @@ def __init__(self, dir_, pipeline=None):
                                self.graph)
 
     def _init_first_time(self, pipeline):
+        """Create the needed subdirectories in .yax"""
         if pipeline is None:
```

Comment:
Someone once told me that evaluating for None was a bad idea... you seem to do it a couple of times in this area.